### DISCUSSION TITLE
Versioned Docker image tag

### DISCUSSION THREAD

**Discussion by gajus** [dragonflydb/dragonfly#2141] [Q&A] [answered]
All documentation references show `docker.dragonflydb.io/dragonflydb/dragonfly` as a Docker image reference.

Is there a versioned tag?

**Comment by andydunstall** [comment]
The release image uses the version as the tag, such as `docker.dragonflydb.io/dragonflydb/dragonfly:v1.12.1`

**Reply by gajus** [reply]
Where are these listed? I feel like I looked everywhere... I would expect some mention at least here https://www.dragonflydb.io/docs/getting-started/docker

**Reply by romange** [reply, marked as answer]
You are right, we will mention them there.
